src/mem/cache/prefetch/bop.cc

```diff
@@ -114,22 +111,24 @@ BOP::delayQueueEventWrapper()
 }
 
 unsigned int
-BOP::hash(Addr addr, unsigned int way) const
+BOP::index(Addr addr, unsigned int way) const
```

Comment:
Nit: in general it is better to avoid these modifications that don´t really change anything, as this could needlessly create merge conflicts for other users. There is no need to revert, though, as it is quite unlikely that someone is modifying the prefetcher.

Reply:
Fixed in the latest commit